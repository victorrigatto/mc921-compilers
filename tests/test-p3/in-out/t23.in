void main() {
  while(1) {
    i=i+1;
  }
}
