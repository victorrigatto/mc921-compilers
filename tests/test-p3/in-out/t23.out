SemanticError: conditional expression is type(int), not type(bool) @ 2:3
